testingutils/src/main/java/com/github/awsjavakit/testingutils/networking/CustomResponseInfo.java

```diff
@@ -8,23 +8,15 @@
 import java.net.http.HttpHeaders;
 import java.net.http.HttpResponse;
 
-public final class CustomResponseInfo implements HttpResponse.ResponseInfo {
-
-  private final int status;
-  private final HttpHeaders httpHeaders;
-
-  public CustomResponseInfo(int status, HttpHeaders httpHeaders) {
-    this.status = status;
-    this.httpHeaders = httpHeaders;
-  }
+@JacocoGenerated
+public record CustomResponseInfo(int status, HttpHeaders httpHeaders) implements HttpResponse.ResponseInfo {
 
   public static CustomResponseInfo create(Response wiremockResponse) {
     return new CustomResponseInfo(wiremockResponse.getStatus(),
       wiremockHeadersToJavaHeaders(wiremockResponse.getHeaders()));
   }
 
   @Override
-  @JacocoGenerated
   public int statusCode() {
     return status;
   }
```

Comment:
This can be deleted

Reply:
It is required from the `implements HttpResponse.ResponseInfo`